test/test_bot.py

```diff
@@ -6,204 +6,177 @@
 
 import pytest
 
-from sopel import bot, config, plugins, trigger
-from sopel.test_tools import MockIRCBackend, rawlist
+from sopel import bot, plugins
+from sopel.tests import rawlist
 
 
 @pytest.fixture
-def tmpconfig(tmpdir):
-    conf_file = tmpdir.join('conf.ini')
-    conf_file.write("\n".join([
+def tmpconfig(configfactory):
+    return configfactory('test.cfg', "\n".join([
         "[core]",
         "owner = testnick",
         "nick = TestBot",
         "enable = coretasks"
```

Comment:
This slipped by a prior code review, I guess.

```suggestion
        "enable = coretasks",
```

It'll work just fine with no comma, obviously, but the fake config file won't have a trailing newline as intended.

(GitHub is showing me that I removed a leading space in my suggestion, but I did no such thing and there are indeed 8 spaces still. Must be already Friday afternoon in the GitHub datacenter. 😹)